src/components/project-details/project-details.tsx

```diff
@@ -4,18 +4,38 @@
  */
 
 import { Project } from 'utils';
-import './project-details.module.css';
+import styles from './project-details.module.css';
 
 interface ProjectDetailsProps {
   project: Project;
 }
 
 const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }: ProjectDetailsProps) => {
   return (
-    <div>
+    <div id={styles.project_details}>
```

Comment:
Any reason why `id` instead of `className`?

Reply:
I read that id was used for unique elements. Is it better to use class in this case?

Reply:
Would you be able to send me the link where you read that? I'd love to learn more about why. If not, that's okay, just let me know.

Reply:
https://css-tricks.com/the-difference-between-id-and-class/

Reply:
Thank you! I actually didn't realize URLs with `...#some-tag` were done via HTML element IDs. Learn something new every day.

In this case, it's a little nuanced. You should use both an ID and a class. Use the ID for styling that is unique to the element and so that we can uniquely identify it on the page. Use the class for any styling that might end up being shared between other tiles on the page that you can see in the design sketch snippet that is attached to the issue #73.